tests.py

```diff
@@ -179,7 +158,7 @@ def test_copy_files(self):
                 'cp a %s' % n('/dest/dir/a'),
                 'mkdir %s' % n('/dest/dir/b'),
                 'makedirs %s' % n('/dest/dir/c/d'),
-                'cp %s %s' % (n('c/d/e'), n('/dest/dir/c/d/e')),
+                'cp {} {}'.format(n('c/d/e'), n('/dest/dir/c/d/e')),
```

Comment:
Okay, why change this line but leave all of the previous lines using %-formatting?  This offends my sense of consistency.